nexteuropa_newsroom.admin.inc

```diff
@@ -221,7 +221,7 @@ function nexteuropa_newsroom_admin_settings($form, &$form_state) {
   $form['newsroom_universe']['nexteuropa_newsroom_allowed_ips'] = [
     '#type' => 'textfield',
     '#title' => t('IP addresses allowed for import'),
-    '#default_value' => variable_get('nexteuropa_newsroom_allowed_ips', '127.0.0.1'),
+    '#default_value' => variable_get('nexteuropa_newsroom_allowed_ips', '127.0.0.1,158.167.165.192,158.167.165.191,158.167.209.222,158.167.240.147,158.167.209.223,158.167.210.159,158.167.210.158'),
```

Comment:
1) These IP addresses provided by administrators of the servers. I don't think this will be useful and it will help some how users of newsroom connector. Since very beginning we didn't updated it. So, normal users will not updated this information.  Besides, that it would take a lot of time before we get precise information about every IP address.
2) I don't see a point to create unnecessary complexity and convert string to array here. It works perfectly as string. The solution with array will not be faster, it will just add additional complexity, but if you insist let's do it in the next version.